test_scripts/API/ButtonSubscription/002_Subscribe_CUSTOM_BUTTON_Success.lua

```diff
@@ -0,0 +1,35 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check that SDL processes SubscribeButton RPC with 'CUSTOM_BUTTON' parameter
+------------------------------------------------------------------------------------------------------------------------
+-- In case:
+-- 1) Mobile app starts registration
+-- SDL does:
+-- 1) Send Buttons.SubscribeButton(CUSTOM_BUTTON, appId) to HMI during registration
+-- 2) Wait response from HMI
+-- 3) Receive Buttons.SubscribeButton(SUCCESS)
+-- 4) Not send OnHashChange with updated hashId to mobile app
+------------------------------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local buttonName = "CUSTOM_BUTTON"
+
+--[[ Scenario ]]
+common.runner.Title("Preconditions")
+common.runner.Step("Clean environment", common.preconditions)
+common.runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
```

Comment:
add button to HMI capabilities during start HMI:
common.runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { common.addButtonToCapabilities(common.custom_button_obj) })

Reply:
addButtonToCapabilities implemented in common_buttonSubscription.lua in PR https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/127